flycheck-haskell.el

```diff
@@ -143,6 +143,9 @@ Return the configuration."
   (or (flycheck-haskell-get-cached-configuration cabal-file)
       (flycheck-haskell-read-and-cache-configuration cabal-file)))
 
+(defconst flycheck-haskell-cabal-config "cabal.config"
```

Comment:
Shouldn't we look at `cabal.sandbox.config` as well?  Currently it doesn't contain this setting, but it probably might do so in future.